include/api.php

```diff
@@ -1556,22 +1557,31 @@ function api_search($type)
 		}
 		$terms = DBA::select('term', ['oid'], $condition, []);
 		$itemIds = [];
-		while($term = DBA::fetch($terms)){ $itemIds[] = $term['oid']; }
+		while ( $term = DBA::fetch($terms) ) {
+			$itemIds[] = $term['oid'];
+		}
 		DBA::close($terms);
-		$condition = [$exclude_replies ? "`id` = `parent` AND " : ''];
-		$condition[0] .= empty($itemIds) ? '' : ' `id` IN ('.implode(", ", $itemIds).')' ;
 
+		if (empty($itemIds)) {
+			return api_format_data("statuses", $type, $data);
+		}
+
+		$tmpAr = ['`id` IN ('.implode(", ", $itemIds).')'];
+		if ($exclude_replies) {
+			$tmpAr[] = "`id` = `parent`";
+		}
+
+		$condition = [ implode(" AND ", $tmpAr) ];
```

Comment:
Code standards: Please remove spaces inside square brackets and use single quotes for strings.